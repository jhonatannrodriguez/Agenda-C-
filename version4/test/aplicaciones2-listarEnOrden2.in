# se asume un ID mÃ¡ximo de persona de 10

crearTTablaPersonas 20

# creo Persona 3 y agrego a tabla
crearAgenda
crearPersona 3 30 Persona3
crearFecha 3/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 4 y agrego a tabla
crearAgenda
crearPersona 4 40 Persona4
crearFecha 4/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 5 y agrego a tabla
crearAgenda
crearPersona 5 50 Persona5
crearFecha 5/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 6 y agrego a tabla
crearAgenda
crearPersona 6 60 Persona6
crearFecha 6/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 7 y agrego a tabla
crearAgenda
crearPersona 7 70 Persona7
crearFecha 7/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 8 y agrego a tabla
crearAgenda
crearPersona 8 80 Persona8
crearFecha 8/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 9 y agrego a tabla
crearAgenda
crearPersona 9 90 Persona9
crearFecha 9/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 10 y agrego a tabla
crearAgenda
crearPersona 10 10 Persona10
crearFecha 10/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla


# el orden debe ser 3, 4, 5, 6, 7, 10
listarEnOrden 6 Persona5 Persona7 Persona6 Persona3 Persona10 Persona4

liberarTTablaPersonas

Fin
