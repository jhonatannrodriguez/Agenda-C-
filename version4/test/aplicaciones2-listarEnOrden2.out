1>#  se asume un ID máximo de persona de 10.
2>3>#  creo Persona 3 y agrego a tabla.
4>5>6>7>8>9>10>#  creo Persona 4 y agrego a tabla.
11>12>13>14>15>16>17>#  creo Persona 5 y agrego a tabla.
18>19>20>21>22>23>24>#  creo Persona 6 y agrego a tabla.
25>26>27>28>29>30>31>#  creo Persona 7 y agrego a tabla.
32>33>34>35>36>37>38>#  creo Persona 8 y agrego a tabla.
39>40>41>42>43>44>45>#  creo Persona 9 y agrego a tabla.
46>47>48>49>50>51>52>#  creo Persona 10 y agrego a tabla.
53>54>55>56>57>58>59>#  el orden debe ser 3, 4, 5, 6, 7, 10.
60>Persona 3: Persona3, 30 años
Evento 1:  Evento1
Fecha: 3/10/2022
Persona 4: Persona4, 40 años
Evento 1:  Evento1
Fecha: 4/10/2022
Persona 5: Persona5, 50 años
Evento 1:  Evento1
Fecha: 5/10/2022
Persona 6: Persona6, 60 años
Evento 1:  Evento1
Fecha: 6/10/2022
Persona 7: Persona7, 70 años
Evento 1:  Evento1
Fecha: 7/10/2022
Persona 10: Persona10, 10 años
Evento 1:  Evento1
Fecha: 10/10/2022
61>62>Fin.
